<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="lefthalf">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_fcfs">
       <attribute name="title">
        <string>FCFS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="gantt_fcfs"/>
        </item>
        <item>
         <widget class="QTableWidget" name="results_fcfs">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="avg_fcfs">
          <row>
           <property name="text">
            <string>average value</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_priority">
       <attribute name="title">
        <string>Priority</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGraphicsView" name="gantt_priority">
          <property name="maximumSize">
           <size>
            <width>441</width>
            <height>91</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="results_priority">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="avg_priority">
          <row>
           <property name="text">
            <string>average value</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_pp">
       <attribute name="title">
        <string>PP</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGraphicsView" name="gantt_pp"/>
        </item>
        <item>
         <widget class="QTableWidget" name="results_pp">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="avg_pp">
          <row>
           <property name="text">
            <string>average value</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_rr">
       <attribute name="title">
        <string>RR</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGraphicsView" name="gantt_rr"/>
        </item>
        <item>
         <widget class="QTableWidget" name="results_rr">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="avg_rr">
          <row>
           <property name="text">
            <string>average value</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_pprr">
       <attribute name="title">
        <string>PRR</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGraphicsView" name="gantt_pprr"/>
        </item>
        <item>
         <widget class="QTableWidget" name="results_pprr">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="avg_pprr">
          <row>
           <property name="text">
            <string>average value</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_sjf">
       <attribute name="title">
        <string>SJF</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGraphicsView" name="gantt_sjf"/>
        </item>
        <item>
         <widget class="QTableWidget" name="results_sjf">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="avg_sjf">
          <row>
           <property name="text">
            <string>average value</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_srtf">
       <attribute name="title">
        <string>SRTF</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGraphicsView" name="gantt_srtf"/>
        </item>
        <item>
         <widget class="QTableWidget" name="results_srtf">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>pid</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="avg_srtf">
          <row>
           <property name="text">
            <string>average value</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>response time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>turnaround time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>waiting time</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="righthalf">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_timeslice">
          <property name="text">
           <string>Timeslice: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_timeslice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Timeslice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_run">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteall">
          <property name="text">
           <string>Delete All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_processes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>520</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>520</width>
          <height>16777215</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>pid</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>arrival time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>burst time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>priority</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="layout_label">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label_pid">
              <property name="text">
               <string>pid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_arrival">
              <property name="text">
               <string>arrival</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_burst">
              <property name="text">
               <string>burst</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_priority">
              <property name="text">
               <string>priority</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_lineEdit">
            <item>
             <widget class="QLineEdit" name="lineEdit_pid">
              <property name="text">
               <string>pid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_arrival">
              <property name="text">
               <string>arrival</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_burst">
              <property name="text">
               <string>burst</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_priority">
              <property name="text">
               <string>priority</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
